src/utils/ssmCommand.ts

```diff
@@ -8,26 +8,20 @@ const s3 = new AWS.S3();
 export function ssmCommand(command: string, instanceId: string, outputExpected: boolean = true): Promise<string> {
     return new Promise<string>((resolve, reject) => {
         sendCommand(command, instanceId)
-            .then((sendResult: SSM.Types.SendCommandResult) => wait(10000, sendResult.Command.CommandId))
-            .then((commandId: string) => getCommandResult(commandId, instanceId))
+            .then(delay)
```

Comment:
Minor suggestion: now that we have polling we could drop this initial delay (since waiting should now be handled within `awaitCommandResult`)

Reply:
We've decided to keep this delay, just to make sure we give AWS a chance to process the `sendCommand` function before we ask for the `CommandId`